pkgs/by-name/vo/voidmap/package.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage {
+  pname = "voidmap";
+  version = "1.1.5-unstable-2023-09-13";
+
+  src = fetchFromGitHub {
+    owner = "void-rs";
+    repo = "void";
+    rev = "ab32290632fa9477a7e59b884bdfa69fb4b91906";
+    hash = "sha256-+P83psu+BYzgC/I/Ul7vrbZ99KIybd410/ycsIY1pGI=";
+  };
+
+  cargoHash = "sha256-3pObVbtvT6rL7JZ3DyTWenVog5WdZno1BcgKt+WelFs=";
+
+  doCheck = false;
```

Comment:
Upstream seems to have added tests. Please explain the reason for disabling tests in a comment near this line.

Reply:
I moved to skipping the offending test. The test case utilizes file io to clone the console stdout to a file with dup2, so I suspect it is the sandbox expectations that were causing the fail. I haven't packaged a Rust application before so if you believe it's worth the effort to patch the test vs skipping the offending `qc_input_events_dont_crash_void` I'm happy to do so.